classes/controller/FrontController.php

```diff
@@ -1527,6 +1527,7 @@ public function getTemplateVarConfiguration()
         return array(
             'display_taxes_label' => $this->getDisplayTaxesLabel(),
             'display_prices_tax_incl' => (bool) (new TaxConfiguration())->includeTaxes(),
+            'tax_is_active' => (bool) Configuration::get('PS_TAX'),
```

Comment:
Can we change this by "taxes_enabled"?

Reply:
Yes sir ! https://github.com/PrestaShop/PrestaShop/pull/13047/commits/6ffbd08ed99eecd810eb5992c8e788a08f81775c